pkg/controller/operator/seed/reconciler.go

```diff
@@ -576,28 +608,31 @@ func (r *Reconciler) reconcileServices(ctx context.Context, cfg *kubermaticv1.Ku
 }
 
 func (r *Reconciler) reconcileAdmissionWebhooks(ctx context.Context, cfg *kubermaticv1.KubermaticConfiguration, seed *kubermaticv1.Seed, client ctrlruntimeclient.Client, log *zap.SugaredLogger) error {
-	log.Debug("reconciling AdmissionWebhooks")
+	log.Debug("reconciling Admission Webhooks")
 
 	validatingWebhookCreators := []reconciling.NamedValidatingWebhookConfigurationCreatorGetter{
 		common.SeedAdmissionWebhookCreator(cfg, client),
 		kubermaticseed.ClusterValidatingWebhookConfigurationCreator(cfg, client),
 	}
 
 	if cfg.Spec.FeatureGates[features.OperatingSystemManager] {
-		validatingWebhookCreators = append(validatingWebhookCreators, kubermaticseed.OperatingSystemProfileValidatingWebhookConfigurationCreator(cfg, client))
-		validatingWebhookCreators = append(validatingWebhookCreators, kubermaticseed.OperatingSystemConfigValidatingWebhookConfigurationCreator(cfg, client))
+		validatingWebhookCreators = append(
+			validatingWebhookCreators,
+			kubermaticseed.OperatingSystemProfileValidatingWebhookConfigurationCreator(cfg, client),
+			kubermaticseed.OperatingSystemConfigValidatingWebhookConfigurationCreator(cfg, client),
+		)
 	}
 
 	if err := reconciling.ReconcileValidatingWebhookConfigurations(ctx, validatingWebhookCreators, "", client); err != nil {
-		return fmt.Errorf("failed to reconcile validating AdmissionWebhooks: %w", err)
+		return fmt.Errorf("failed to reconcile validating Validating Webhooks: %w", err)
```

Comment:
Second nit: _"[...] validating Validating Webhooks"_. The previous error was probably fine ("validating AdmissionWebhooks" is fairly accurate).